src/navigate-pop/switching-apps.md

```diff
@@ -0,0 +1,56 @@
+# Navigate Between Application Windows
+
+Pop!\_OS uses the GNOME desktop environment, which includes keyboard shortcuts for quickly switching between system and application windows.
+
+---
+
+## Using the Dock
+
+The Dock will will designate a running application with a small white dot. Click the application to bring it's window into focus.
```

Comment:
"It's" is _always_ short for "it is." The possessive is _always_ "its" with no apostrophe.

Reply:
I'm aware, just human. Good catch.